src/pages/health-declaration/health-declaration.scss

```diff
@@ -26,3 +26,24 @@ page-health-declaration {
     }
   }
 }
+.confirm-declaration-modal {
+  .alert-head:before {
+    display: inline-block;
+    font-size: 68px;
+    content: '\f276';
```

Comment:
What's this `content` about?

Reply:
it's the ! icon at the top of the alert

Reply:
I copied this from another alert.  It is a special code to display the exclamation mark at the top of the alert.